python/feathub/processors/flink/table_builder/tests/test_aggregation_utils.py

```diff
@@ -0,0 +1,86 @@
+#  Copyright 2022 The FeatHub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import unittest
+from datetime import timedelta
+
+from pyflink.table import DataTypes
+
+from feathub.common.exceptions import FeathubException
+from feathub.common.types import Int32
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.transforms.over_window_transform import OverWindowTransform
+from feathub.feature_views.transforms.sliding_window_transform import (
+    SlidingWindowTransform,
+)
+from feathub.processors.flink.table_builder.aggregation_utils import (
+    AggregationFieldDescriptor,
+)
+
+
+class AggregationUtilsTest(unittest.TestCase):
```

Comment:
Can you also add an integration test shows the benefit of this change for a Python program? For example, we should be able to show that when filter expression is used with OverWindow, the output column will no longer contain null value for rows that do not match the filter expression.

There is a request for this feature in the DingDing user group. I will copy/paste the message from user below:

```
1.譬如算 '用户最近1小时点击次数'， 
Feature(
        name="click_count",
        transform=OverWindowTransform(
            expr="bhv_type",
            agg_func="COUNT",
            window_size=timedelta(hours=1),
            group_by_keys=["user_id"],
            limit=None,
            filter_expr="bhv_type='click'"
        ),
    )
在 bhv_type不等于click时，赋null值，如果 该用户最近1小时有发生点击，则取null值不符合该特征的含义。
2.另外 A. group_by_keys 在窗口范围内没有事件会取null 值; B. filter_expr not match 取null值，这两种情况都会取 null值，导致没法区分

```

Reply:
Sounds good.